drafts/2020-04-09-release-20004.adoc

```diff
@@ -0,0 +1,690 @@
+=== Publish a standalone Open Liberty Grafana dashboard to grafana.com
+
+The Grafana dashboard provides a wide range of time-series visualizations of MicroProfile Metrics data such as CPU, Servlet, Connection Pool, and Garbage Collection metrics. It is powered by a Prometheus datasource which is configured to ingest data from one or more Liberty servers' `/metrics` endpoint, enabling us to view on Grafana in near real-time.
+
+Previously, support was only provided for visualizing mpMetrics data via Grafana in Open Liberty servers running on Kubernetes environments such as the OpenShift Container Platform. The new Grafana dashboard can display `mpMetrics` data on standalone Open Liberty servers, meaning anybody can utilize the add-on by starting up an Open Liberty server with `mpMetrics-2.x` enabled on their localhost alongside Grafana and Prometheus.
+
+You can benefit from this dashboard while testing new apps to deploy on Open Liberty. For instance, metrics such as servlet response times, CPU or heap usage when seen as a time-series on Grafana, could be indicative of an underlying performance issue or memory leak.
+
+To configure the dashboard, first add the `mpMetrics-2.3` feature. This would automatically enable the `monitor-1.0` feature: 
+
+[source]
+----
+<featureManager>
+    <feature>mpMetrics-2.3</feature>
+</featureManager>
+
+<mpMetrics authentication="false" />`
+----
+
+For metrics on a secure endpoint:
+[source]
+----
+<featureManager>
+    <feature>mpMetrics-2.3</feature>
+</featureManager>
+
+<quickStartSecurity userName="<your-username>" userPassword="<your-password>" />
+----
+
+Run the server using the following command: 
+
+[source]
+----
+./server run DashboardTest
+----
+
+Then, download https://prometheus.io/download/[Prometheus]. Once unpackaged, it should contain a startup script called prometheus alongside a configuration file, `prometheus.yml`. Within `prometheus.yml`, append to `scrape_configs` one of the following jobs/
+
+For `mpMetrics` on an insecure endpoint:
+[source]
+----
+- job_name: 'liberty'
+    scrape_interval: 5s
+    static_configs:
+      - targets: ['localhost:9080']
+----
+
+For `mpMetrics` on a secure endpoint:
+[source]
+----
+- job_name: 'liberty-secure'
+    scrape_interval: 5s
+    static_configs:
+      - targets: ['localhost:9443']
+    basic_auth:
+      username: "<your-username>"
+      password: "<your-password>"
+    tls_config:
+       insecure_skip_verify: true
+    scheme: "https"
+----
+
+Start the prometheus script and visit http://localhost:9090/targets, where you should see your Open Liberty server listed as one of the targets.
+
+image::img/blog/20004-prometheus.png[align="center"]
+
+Download https://grafana.com/docs/grafana/latest/installation/[Grafana] and once finished, visit https://localhost:3000.
+
+When navigated to Grafana, click the gear icon and select "Data Source". Add a new Prometheus datasource with the URL as `\http://localhost:9090`. Then click "Save & Test".
```

Comment:
For these actions where they are told to click something, can you make them **bold** instead of in quotation marks?